tests/matchers_unittest.py

```diff
@@ -131,7 +131,11 @@ def testAny(self):
 
     def testAnyInstanceOf(self):
         self.assertEqual(testslide.matchers.AnyInstanceOf(str), "durrdurr")
+
+    def testTriggeringAnyInstanceOfException(self):
```

Comment:
nit. I would suggest to keep a single test just to keep it uniform with the rest of the tests

Reply:
Okay, I believe the `self.assertNotEqual(testslide.matchers.AnyInstanceOf(str), 7)` was unintentionally selected to be added to the triggering test.

I'll keep it where it was initially, in `testAnyInstanceOf`.
And only add a single triggerTest to `testTriggeringAnyInstanceOfException`.

Is it what I am expected to do?